library/src/main/java/com/schibsted/spain/barista/BaristaClickActions.java

```diff
@@ -29,6 +29,15 @@
   private static final ResourceTypeChecker RESOURCE_TYPE_CHECKER = new ResourceTypeChecker();
 
   // SingleClick Actions
+  public static void clickOn(int id) {
+    performWithResource(id, ViewActions.click());
+  }
+
+  public static void clickOn(String text) {
+    performWithMatcher(withText(text), ViewActions.click());
+  }
+
+  // Deprecated SingleClick Actions
```

Comment:
You should use the deprecated annotation to tell consumers what's going on.
Also, if this goes into Barista 2.0 I wonder if we should deprecate or simply replace the method breaking the API.